spk/jackett/Makefile

```diff
@@ -30,23 +31,31 @@ UNSUPPORTED_ARCHS = $(PPC_ARCHS)
 
 # Mono build for unsupported .NET archs
 include ../../mk/spksrc.archs.mk
-ifeq ($(findstring $(ARCH),$(ARMv5_ARCHS) $(i686_ARCHS) $(ARMv7L_ARCHS)),$(ARCH))
+include ../../mk/spksrc.common.mk
+ifeq ($(call version_lt, ${TCVERSION}, 7.0),1)
+ifeq ($(findstring $(ARCH),$(ARMv5_ARCHS) $(i686_ARCHS) $(ARMv7L_ARCHS) $(ARMv7_ARCHS)),$(ARCH))
 DEPENDS = cross/curl cross/$(SPK_NAME)-mono
 SERVICE_SETUP = src/service-setup-mono.sh
-CHANGELOG = "Update Jackett to 0.20.1171"
-LICENSE_FILE = $(WORK_DIR)/Jackett/LICENSE
 SPK_DEPENDS = "mono>3.6"
 endif
+else
+ifeq ($(findstring $(ARCH),$(ARMv5_ARCHS) $(i686_ARCHS) $(ARMv7L_ARCHS) armada370),$(ARCH))
+DEPENDS = cross/curl cross/$(SPK_NAME)-mono
+SERVICE_SETUP = src/service-setup-mono.sh
+SPK_DEPENDS = "mono>3.6"
+endif
+endif
 
 POST_STRIP_TARGET = jackett_extra_install
 
 # use alternate TMPDIR as /tmp might be too small and not accessible on DSM >= 7.1.
+# TMPDIR is used for jackett built in backup and update
 USE_ALTERNATE_TMPDIR = 1
 
 include ../../mk/spksrc.spk.mk
 
 .PHONY: jackett_extra_install
 jackett_extra_install:
-	install -m 755 -d $(STAGING_DIR)/var
-	install -m 755 -d $(STAGING_DIR)/var/.config
-	install -m 755 -d $(STAGING_DIR)/var/.config/Jackett
+	@install -m 755 -d $(STAGING_DIR)/var
+	@install -m 755 -d $(STAGING_DIR)/var/.config
```

Comment:
those lines are obsolete, install -d creates the whole path (like mkdir -p does)
```suggestion
```